modules/local/genome_uploader/webin_cli_upload/main.nf

```diff
@@ -32,15 +32,20 @@ process WEBIN_CLI_UPLOAD {
       -submit \
       ${mode}
 
-    if grep -q "submission has been completed successfully" webin-cli.report; then
-        export SUCCESS_STATUS="true"
-    else
-        export SUCCESS_STATUS="false"
-    fi
+    mv webin-cli.report "${id}_webin-cli.report"
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         ena-webin-cli: \$(ena-webin-cli -version 2>&1 )
     END_VERSIONS
+
+    # status check
+    if grep -q "submission has been completed successfully" "${id}_webin-cli.report"; then
+        export SUCCESS_STATUS="true"
```

Comment:
One tiny suggestion.. maybe instead of "true"/"false" use "success" - "failed". The true/false version looks weird as we are comparing strings.